### PR TITLE
[BUGFIX][DOCS] 0.13 release fixes

### PR DESCRIPTION
**Pull request by roblim**
Changes proposed in this pull request:
- Fix missing _validator_config bug
- Fix interactive_validation flag behavior in Validator
- Fix bug in run_validation_operator related to retrieving batch_id from batch or validator
- Fix data docs rendering bug (case where batch_spec is missing)
- Docs fixes: add link to custom renderer how-to in custom expectation doc
- Update all validate_configuration methods to support eval params

### CODE REVIEW

**Comment by abegong** [comment]
@roblim , these changes look great. I suspect they'll also resolve some of the issues I'm running into as I connect the dots between Batches and Profilers.

Are you planning to include tests in this PR? If you do, I should be able to review for you.

File: docs/guides/how_to_guides/spare_parts/how_to_create_custom_expectations.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. _how_to_guides__creating_and_editing_expectations__how_to_create_custom_expectations:
+.. _how_to_guides__creating_and_editing_expectations__how_to_create_custom_expectations_sp:
```
Comment (Shinnnyshinshin) [inline docs/guides/how_to_guides/spare_parts/how_to_create_custom_expectations.rst:1]: just curious. what is the role of the `_sp` at the end?

File: great_expectations/validation_operators/validation_operators.py [hunk 2]
```diff
@@ -313,11 +313,17 @@ def run(
...
+                batch_identifier = batch.active_batch_id
+            else:
+                batch_identifier = batch.batch_id
+
             expectation_suite_identifier = ExpectationSuiteIdentifier(
                 expectation_suite_name=batch._expectation_suite.expectation_suite_name
             )
             validation_result_id = ValidationResultIdentifier(
-                batch_identifier=batch.batch_id,
+                batch_identifier=batch_identifier,
```
Comment (Shinnnyshinshin) [inline great_expectations/validation_operators/validation_operators.py:326]: much cleaner :) thank you

**Review by Shinnnyshinshin — CHANGES_REQUESTED** [review]
Hi @roblim this looks wonderful :)

I wanted to second @abegong comment regarding tests (or do they already exist in the codebase?)

Also, I the Azure pipelines seem to be failing. Maybe you could have a look.

File: docs/guides/how_to_guides/spare_parts/how_to_create_custom_expectations.rst (hunk 1 above)
Comment (roblim) [reply docs/guides/how_to_guides/spare_parts/how_to_create_custom_expectations.rst:1]: There was a name collision, so I added "sp" to denote this as "spare parts".

**Comment by roblim** [comment]
> @roblim , these changes look great. I suspect they'll also resolve some of the issues I'm running into as I connect the dots between Batches and Profilers.
> 
> Are you planning to include tests in this PR? If you do, I should be able to review for you.

Sure, I can add some tests.

**Comment by roblim** [comment]
Thanks for pushing for those tests @abegong - ended up finding a bunch of broken stuff with eval param substitution. Ready for re-review.

**Comment by Shinnnyshinshin** [comment]
wonderful thank you :)

**Review by Shinnnyshinshin — APPROVED** [review]

**Pull request merged**